patches/node/build_add_gn_build_files.patch

```diff
@@ -944,7 +944,25 @@ index 0000000000000000000000000000000000000000..78dff9e0458aaa6b864259f3001764c7
 +    {
 +      "dest_dir": "include/node//cppgc/",
 +      "files": [
-+        "//v8/include/cppgc/finalizer-trait.h"
++        "//v8/include/cppgc/allocation.h",
++        "//v8/include/cppgc/garbage-collected.h",
++        "//v8/include/cppgc/heap.h",
++        "//v8/include/cppgc/member.h",
++        "//v8/include/cppgc/platform.h",
++        "//v8/include/cppgc/source-location.h",
++        "//v8/include/cppgc/trace-trait.h",
++        "//v8/include/cppgc/type-traits.h",
++        "//v8/include/cppgc/visitor.h"
++      ]
++    },
++    {
++      "dest_dir": "include/node//cppgc/internal/",
++      "files": [
++        "//v8/include/cppgc/internal/accessors.h",
```

Comment:
Why are we exposing internal headers through `node_headers` ?

Reply:
The CL commit says,

```
cppgc: Move implementation headers in internal/ dir

Headers containing only implementation details are better to stay in a
separate dir to indicate the user that they shouldn't be included.
```

Reply:
@deepak1556 hmm - i just ran `python tools/generate_gn_filenames_json.py` from the node dir

Reply:
@deepak1556 the non-internal headers include the internal ones. it just means "shouldn't be included _directly_".

Reply:
Okay in that case there is nothing we can do, I was hoping the non-internal ones are like `v8.h`